lib/jsender/json.rb

```diff
@@ -0,0 +1,30 @@
+require 'json'
+
+module Jsender
+  module Json
+    extend self
+
+    def error(message: "An error has occurred")
```

Comment:
Nice clarity in the code. Much simpler to read

Reply:
Thanks, I wonder how we resolve comments like in Gitlab here?